frontend/src/pages/Dashboard.js

```diff
@@ -202,8 +214,20 @@ const Dashboard = () => {
                     p="16px 24px"
                 >
                     <Heading mb="16px" fontSize="3xl">
-                        {`${currentTicket.type}-${currentTicket.id}: ${ticketData.name}`}
+                        {`${currentTicket.codename}`}
                     </Heading>
+                    <Flex flexDir="row" mb="12px">
+                        {/* Do not display delete button for WATO Cash */}
+                        {currentTicket.data.sf_link !== -1 && (
+                            <Button
+                                size="sm"
+                                colorScheme="red"
+                                onClick={onOpenDeleteTicket}
```

Comment:
while making it pretty is not our immediate priority, I don't think there's harm in getting the layout right. I would move this to the right side of the codename heading

Reply:
this is what it looks like on JIRA